packages/browser/src/Component/Itowns/AddLayerFromConfig.js

```diff
@@ -20,7 +20,7 @@ const $3DTemporalTileset = Widget.$3DTemporalTileset;
  * @param {itowns.View} itownsView - the itowns view
  * @returns {itowns.C3DTilesLayer} A 3D Tiles Layer
  */
-function setup3DTilesLayer(layer, layerManager, itownsView) {
+export function setup3DTilesLayer(layer, layerManager, itownsView) {
```

Comment:
there is a case where you have to use this function instead of add3DTilesLayers ?

Reply:
Yes, I want to be able to add a 3DTiles layer without a config file / dynamically

Reply:
:+1: